contribs/gnodev/Makefile

```diff
@@ -5,11 +5,13 @@ GOTEST_FLAGS ?= $(GOBUILD_FLAGS) -v -p 1 -timeout=5m
 rundep := go run -modfile ../../misc/devdeps/go.mod
 golangci_lint := $(rundep) github.com/golangci/golangci-lint/cmd/golangci-lint
 
-install:
+install: install.gnodev
+install.gnodev:
 	go install $(GOBUILD_FLAGS) ./cmd/gnodev
 
-build:
-	go build $(GOBUILD_FLAGS) -o build/gnodev ./cmd/gnodev
+# keep gnobro out the default install for now
+install.gnobro:
```

Comment:
the removal of `build` from Makefile is on purpose?

Reply:
Yes, it was pretty useless as we already have `go install`. I think `go build` should be a manual action from the user.

Reply:
I suggest we keep it. It makes sense to maintain the `make build` command as an alternative documentation option for users who may not have a fully configured development environment or prefer not to modify their system (i.e., for security reasons).